README.md

```diff
@@ -124,6 +126,48 @@ The schedule of the garbage collection, there can always only be one.
 }
 ```
 
+### retention-policies.json
+
+Definition of the retention policies.
+The retention policies can be set per project.
+They can be found in each project page under the tab Policy.
+
+```json
+[
+  {
+    "scope": {
+      "level": "project",
+      "ref": 2
+    },
+    "rules": [
+      {
+        "template": "always",
+        "tag_selectors": [
+          {
+            "decoration": "matches",
+            "pattern": "master"
+          }
+        ],
+        "scope_selectors":  {
+          "repository": [
+            {
+              "kind": "doublestar",
+              "decoration": "repoMatches",
+              "pattern": "**"
+            }
+          ]
+        }
+      }
+    ],
+    "trigger": {
+      "settings": {
+        "cron": "0 0 0 * * *"
```

Comment:
Just a thought: Maybe we should also add some random/prim number to the mins for this an the other examples which have cron schedules?

Reply:
Yes agree, added.